content/en/blog/_posts/2019-09-23-san-diego-contributor-summit

```diff
@@ -0,0 +1,75 @@
+---
+title: "Contributor Summit San Deigo Registration Open!"
+date: 2019-09-23
+slug: san-deigo-contributor-summit
```

Comment:
This should be `san-diego-contributor-summit`

Reply:
corrected